main.js

```diff
@@ -0,0 +1,150 @@
+var startingPeg = ["5", "4", "3", "2", "1"];
+var startingArr1 = [];
+var startingArr2 = [];
+var startingArr3 = [];
+var originalPeg;
+var playingBoard;
+
+var randomizeArrays = function() {
+    randomNum = Math.floor(Math.random() * 3);
+
+    if (randomNum === 0) {
+        originalPeg = 1;
+        startingArr1 = startingPeg.map(function (element){
+            return element;
+        });
+    } else if (randomNum === 1) {
+        originalPeg = 2;
+        startingArr2 = startingPeg.map(function (element) {
+            return element;
+        });
+    } else if (randomNum === 2) {
+        originalPeg = 3;
+        startingArr3 = startingPeg.map(function (element){
+            return element;
+        });
+    }    
+}
+
+var Board = function (arr1, arr2, arr3) {
+    this.arr1 = arr1;
+    this.arr2 = arr2;
+    this.arr3 = arr3;
+
+} 
+
+Board.prototype.moveDisc = function (pos1, pos2) {
+    var peg1 = this.arr1;
+    var peg2 = this.arr2;
+    var peg3 = this.arr3;   
+    var peg1Value = this.arr1[this.arr1.length - 1];
+    var peg2Value = this.arr2[this.arr2.length - 1];
+    var peg3Value = this.arr3[this.arr3.length - 1];     
+    
+    if (pos1 === 1 && pos2 === 2) {   
```

Comment:
Lines 44 - 82 are pretty difficult to read and tend to violate the DRY principle (don't repeat yourself). Is there a way to simplify?